apps/pocket-id/docker-compose.yml

```
@@ -0,0 +1,42 @@
+version: "3.9"
+services:
+  pocket-id:
+    container_name: pocket-id
+    image: stonith404/pocket-id:v0.8.0
+    restart: unless-stopped
+    environment:
+      - TZ=${TZ}
+      - PUBLIC_APP_URL=${APP_PROTOCOL:-https}://${APP_DOMAIN}
+      - TRUST_PROXY=true
+    volumes:
+      - ${APP_DATA_DIR}/data:/app/backend/data 
```

yamllint

[error] trailing spaces

(trailing-spaces)